PSScripts/New-CosmosDbAccountCollections.ps1

```diff
@@ -208,7 +208,69 @@ foreach ($Collection in $CosmosDbConfiguration.Collections) {
         New-CosmosDbCollection @NewCosmosDbCollectionParameters
     }
     else {
-        # TODO: Can we check and modify if something changes
+
         Write-Verbose "$($ExistingCollection.Id) exists"
+        $SetCosmosDbCollectionParameters = @{
+            Context         = $CosmosDbContext
+            Database        = $CosmosDbConfiguration.DatabaseName
+            Id              = $Collection.CollectionName
+        }
+
+        if ($Collection | Get-Member -Name DefaultTtl) {
+
+            if ($ExistingCollection.defaultTtl -eq $Collection.DefaultTtl) {
+
+                Write-Verbose "Time To Live (TTL) already set to $($ExistingCollection.defaultTtl).  Not updating."
+
+            }
+            elseif ($Collection.DefaultTtl -eq 0) {
+
+                #$SetCosmosDbCollectionParameters['RemoveDefaultTimeToLive'] = $true
+                #Write-Verbose "Removing Time To Live (TTL)."
+                throw "Unable to removing Time To Live (TTL), not currently supported.  Remove manually."
+                ##Bug in CosmosDb PowerShell module - https://github.com/PlagueHO/CosmosDB/issues/287
+
+            }
+            else {
+
+                $SetCosmosDbCollectionParameters['DefaultTimeToLive'] = $Collection.DefaultTtl
+                Write-Verbose "Updating Time To Live (TTL) to $($SetCosmosDbCollectionParameters.DefaultTimeToLive)"  
+                            
+            }
+
+        }
+        else {
+
+            if ($ExistingCollection | Get-Member -Name DefaultTtl) {
+                #$SetCosmosDbCollectionParameters['RemoveDefaultTimeToLive'] = $true
+                #Write-Verbose "Removing Time To Live (TTL)."
+                throw "Unable to removing Time To Live (TTL), not currently supported.  Remove manually."
+                ##Bug in CosmosDb PowerShell module - https://github.com/PlagueHO/CosmosDB/issues/287
+            }
+
+        }
+
+        if($SetCosmosDbCollectionParameters.Count -gt 3) {
+
+            $Result = Set-CosmosDbCollection @SetCosmosDbCollectionParameters
+
+        }
+
+        if ($Result | Get-Member -Name DefaultTtl) {
```

Comment:
If $SetCosmosDbCollectionParameters.Count -eq 3 then this line will error out. Move this inside the above if block